commands/operator-sdk/cmd/new.go

```diff
@@ -80,59 +83,162 @@ const (
 
 func newFunc(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, fmt.Errorf("new command needs 1 argument"))
+		log.Fatal("new command needs 1 argument")
 	}
 	parse(args)
 	mustBeNewProject()
 	verifyFlags()
-	g := generator.NewGenerator(apiVersion, kind, operatorType, projectName, repoPath(), generatePlaybook)
-	err := g.Render()
-	if err != nil {
-		cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to create project %v: %v", projectName, err))
-	}
-	if operatorType == goOperatorType {
+	switch operatorType {
+	case goOperatorType:
+		doScaffold()
 		pullDep()
-		generate.K8sCodegen(projectName)
+	case ansibleOperatorType:
```

Comment:
Should we normalize the constants for the operator types? We also have this:

https://github.com/operator-framework/operator-sdk/blob/refactor/rebase-master-add-AO-commands/commands/operator-sdk/cmd/cmdutil/util.go#L31-L36

Reply:
So I think one is for the user(command line input) and one is for internal use when determining if ansible operator vs golang operator during build/up local.

If we think it is really important that these are the same then we need to choose the string versions and we lose the default value functionality of an iota.

Reply:
So I think one is for the user(command line input) and one is for internal use when determining if ansible operator vs golang operator during build/up local.

If we think it is really important that these are the same then we need to choose the string versions and we lose the default value functionality of an iota.

Reply:
It's probably not important for this PR regardless, but something I noticed. Maybe another option is to add a helper/validation function that verifies that the string input is a valid operator type and returns the internal int-based const?